CMake/External_Tensorflow.cmake

```diff
@@ -0,0 +1,90 @@
+# The Tensorflow external project
+
+if(NOT fletch_BUILD_WITH_PYTHON)
+  message(FATAL_ERROR "Error: A build with Python is required for building Tensorflow")
+endif()
+
+if(WIN32)
+  if(NOT fletch_ENABLE_SWIG)
+    message(FATAL_ERROR "Error: SWIG is required for building Tensorflow on Windows")
+  else()
+    set(Tensorflow_DEPENDS ${Tensorflow_DEPENDS} SWIG)
+    if(WIN32)
+      set(SWIG_BUILD_FLAGS
+        -DSWIG_DIR=${fletch_BUILD_PREFIX}/src/SWIG-build
+        -DSWIG_EXECUTABLE=${fletch_BUILD_PREFIX}/src/SWIG-build/swig.exe)
```

Comment:
Use SWIG_ROOT here for generic access. Don't use the one in the swig build directory.